README.md

```diff
@@ -1,98 +1,139 @@
 
-First you need to install Haxe and HaxeFlixel. I'm too lazy to write and keep updated with that setup (which is pretty simple). 
+## What can I do with compiling the game? Why should I compile the game instead of downloading it from [the Itch.io page of the game](https://ninja-muffin24.itch.io/friday-night-funkin)?
+
+Compiling the game gives access to the `/source` folder, allowing you to change the code of the game. You can add a lot of cool things with the open source code!
+
+I recommend having a good idea on how to program. Compiling the game is not for everyone!
+
+If you just want to download and play the game normally, you can click [here to go to the Itch.io page of the game](https://ninja-muffin24.itch.io/friday-night-funkin)!
+
+### **If you do want to compile, continue reading!**
+
+# Installing the Required Programs
+
+![Haxe_Logo](/art/haxeLogo.png)
+
+First you need to install Haxe and HaxeFlixel.
 1. [Install Haxe 4.1.5](https://haxe.org/download/version/4.1.5/) (Download 4.1.5 instead of 4.2.0 because 4.2.0 is broken and is not working with gits properly...)
 2. [Install HaxeFlixel](https://haxeflixel.com/documentation/install-haxeflixel/) after downloading Haxe
 
-Other installations you'd need is the additional libraries, a fully updated list will be in `Project.xml` in the project root. Currently, these are all of the things you need to install:
+Other installations you will need are the additional libraries. A fully updated list will be in `Project.xml` in the project root. Currently, these are all the things you need to install:
 ```
 flixel
 flixel-addons
 flixel-ui
 hscript
 newgrounds
 ```
-So for each of those type `haxelib install [library]` so shit like `haxelib install newgrounds`
+So, for each of those type `haxelib install [library]` so shit like `haxelib install newgrounds`.
 
-You'll also need to install a couple things that involve Gits. To do this, you need to do a few things first.
+You will also need to install a couple things that involve Gits. To do this, you need to do a few things first.
 1. Download [git-scm](https://git-scm.com/downloads). Works for Windows, Mac, and Linux, just select your build.
 2. Follow instructions to install the application properly.
 3. Run `haxelib git polymod https://github.com/larsiusprime/polymod.git` to install Polymod.
 4. Run `haxelib git discord_rpc https://github.com/Aidan63/linc_discord-rpc` to install Discord RPC.
+5. Optional: - Run `haxelib git flixel-addons https://github.com/HaxeFlixel/flixel-addons` to update Flixel-Addons. This fixes the transition bug for zoomed out stage cameras.
 
 You should have everything ready for compiling the game! Follow the guide below to continue!
 
-At the moment, you can optionally fix the transition bug in songs with zoomed out cameras.
-- Run `haxelib git flixel-addons https://github.com/HaxeFlixel/flixel-addons` in the terminal/command-prompt.
-
-### Ignored files
+# Adding `APIStuff.hx` into `/source`

```

Comment:
```suggestion
## Adding `APIStuff.hx` into `/source`
```
This header looks a little bit bigger than it should be.

Reply:
I made it like that because it is in its own category, along with being very important. It is required to compile.